README.md

```diff
@@ -1,6 +1,17 @@
-## GenAI Pipeline Repository
+# OpenVINO™ GenAI
 
-The GenAI repository contains pipelines that implement image and text generation tasks.
+The OpenVINO™ GenAI repository consists of GenAI library and additional GenAI samples.
+
+## OpenVINO™ GenAI Library
+
+OpenVINO™ GenAI is a new flavor of OpenVINO, aiming to simplify running inference of generative AI models.
```

Comment:
```suggestion
OpenVINO™ GenAI is a flavor of OpenVINO, aiming to simplify running inference of generative AI models.
```
It's new only relative to the moment of writing. It will get older as the time passes.

Reply:
Fixed